### ISSUE TITLE
video.js can support dash/hls audio language change?

### ISSUE THREAD

**Issue by friendtasy** [videojs/video.js#8216]
### Description

video.js can support dash/hls audio language change?

### Steps to reproduce

1.video.js support dash/hls audio language change
2.
3.

### What version of Video.js are you using?

8.0.4

### What browser(s) including version(s) does this occur with?

chrome 111

### What OS(es) and version(s) does this occur with?

win11

**Comment by amtins** [comment]
@friendtasy yes, please see the following examples from videojs-http-streaming test page:
- [dash](https://videojs-http-streaming.netlify.app/?debug=false&autoplay=false&muted=false&fluid=false&minified=false&sync-workers=false&liveui=true&llhls=true&url=https%3A%2F%2Fdash.akamaized.net%2Fdash264%2FTestCases%2F10a%2F1%2Fiis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd&type=application%2Fdash%2Bxml&keysystems=&buffer-water=false&exact-manifest-timings=false&pixel-diff-selector=false&network-info=false&dts-offset=false&override-native=true&preload=auto&mirror-source=true) English and German audio
- [hls](https://videojs-http-streaming.netlify.app/?debug=false&autoplay=false&muted=false&fluid=false&minified=false&sync-workers=false&liveui=true&llhls=true&url=https%3A%2F%2Fstorage.googleapis.com%2Fshaka-demo-assets%2Fangel-one-hls%2Fhls.m3u8&type=application%2Fx-mpegurl&keysystems=&buffer-water=false&exact-manifest-timings=false&pixel-diff-selector=false&network-info=false&dts-offset=false&override-native=true&preload=auto&mirror-source=true) multiple audio

**Comment by friendtasy** [comment]
> 

thank you,i want to know how use ffmpeg to deal with two audio language,i have a mp4 file ,the video is h264,have two audio,i use ffmpeg like that ffmpeg -i 11.mp4 -map 0:0 -vcodec copy -map 0:1 -acodec copy -map 0:2 -acodec copy -single_file 1 -adaptation_sets "id=0,streams=v id=1,streams=1，id=2,streams=2" -f dash OUTPUT.mpd,but the auidio belong the same  adaptationset
